tools/testing/test_selections.py

```diff
@@ -271,78 +282,88 @@ def log_time_savings(
     return max_time_savings_sec
 
 
+def _get_file_rating_tests() -> List[str]:
+    path = REPO_ROOT / "test" / TEST_FILE_RATINGS_FILE
+    if not os.path.exists(path):
+        print(f"could not find path {path}")
+        return []
+    with open(path) as f:
+        test_file_ratings = cast(Dict[str, Dict[str, float]], json.load(f))
+    try:
+        changed_files = _query_changed_files()
+    except Exception as e:
+        warn(f"Can't query changed test files due to {e}")
+        return []
+    ratings: Dict[str, float] = {}
```

Comment:
nit: defaultdict would save you from needing the if test_file not in ratings check a couple lines down